tools/state-viewer/src/state_dump.rs

```diff
@@ -196,6 +198,7 @@ pub fn state_dump_redis(
 fn should_include_record(
     record: &StateRecord,
     validators: &HashMap<AccountId, (PublicKey, Balance)>,
+    protocol_treasury_account: &AccountId,
```

Comment:
Instead of passing `validators`, `protocol_treasury_account`, and `select_account_ids`, you can consolidate all these into one argument `select_account_ids` and construct a vector that contains all these three at the call site.

Reply:
Cool, will do!